README.md

```diff
@@ -14,4 +14,4 @@ After adding your libraries and code:
 
 4. Customize the box configuration file (`truffle-box.json`) if necessary.
 
-See [the Truffle Box section of our documentation](https://truffleframework.com/docs/truffle/advanced/creating-a-truffle-box/) for more info.
+See [the Truffle Box section of our documentation](http://trufflesuite.com/docs/truffle/advanced/creating-a-truffle-box.html) for more info.
```

Comment:
Did you mean to make this http?  It does auto-redirect https, but just making sure